CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Breaking changes:
 
 Features:
 
+- [#1248](https://github.com/rails-api/active_model_serializers/pull/1248) Add support for JSON API deserialization (@beauby)
```

Comment:
Well, just point me to however you mark it as 'experimental, don't rely on this' in the code and changelog, and I'm ok with having it merged.  I'd generally like for us to have a way to add experimental features